Explorer/Assets/Scripts/SceneRuntime/Apis/UnityOpsApi.cs

```diff
@@ -22,21 +22,21 @@ public UnityOpsApi(V8ScriptEngine engine, ISceneModuleHub moduleHub, V8Script sc
         }
 
         [UsedImplicitly]
-        public void Log(object message)
+        public void Log(string message)
```

Comment:
Does it fix the problem with printing `object` instead of the content?

Reply:
Nope, its the same. However i've checked that we always receive a `string` in this methods as it is implemented here: https://github.com/decentraland/unity-explorer/blob/97b2196efdfe39ce2c86ba28b1420c0d16d24793/Explorer/Assets/StreamingAssets/Js/Init.js#L39
So i dont see a reason on why keeping the object there